### ISSUE TITLE
DD 0.12

### ISSUE THREAD

**Issue by ryzhyk** [TimelyDataflow/differential-dataflow#270]
It's been a while since DD 0.11 was published on crates.io and I think there have been quite a few API changes and improvements since then. I would love to be able to use the latest version through crates.io.  The part that I am particularly interested in is dogs^3. (we have a DDlog branch that works with the github version of DD, including dogs, but keeping it in sync with the master branch that uses DD from crates.io has become a significant time sink).

Is there any chance we will have a new crates.io release in the near future?

Thanks!

**Comment by frankmcsherry** [comment]
I think dogs^3 has to be publish as its own crate, unless it gets moved in to differential proper (not expected, I think). Is it right that semi-stable crates.io versions of the two crates works for you?

**Comment by ryzhyk** [comment]
Yes, that would totally work, as long as the dogs crate points to a crates.io version of DD and timely. Thank you so much!

DD 0.11 has been very stable for us across lots of tests and real-world use cases. When you say "semi-stable", do you think that there may have been regressions in DD since 0.11 or are you still concerned about dogs not being stable (the latter is fine, as we still consider it experimental feature in DDlog)?

Leonid

**Comment by frankmcsherry** [comment]
Stable in the sense that 0.12 will probably be different from 0.11, and 0.13 will be different again. No expected regressions.

**Comment by ryzhyk** [comment]
Oh, API changes are perfectly fine as far as I'm concerned.

**Comment by antiguru** [comment]
We've resumed publishing Differential (and Timely) to crates. Closing.
